.github/workflows/pull_request.yml

```
@@ -3,7 +3,10 @@ on: pull_request
 name: Check pull request
 jobs:
   test-pr:
-    runs-on: ubuntu-latest
+    runs-on: macos-latest
```

```suggestion
    runs-on: ubuntu-latest
```
I just want to verify whether the incredibly long delay is due this running on macOS.

---

Ubuntu image does not ship with hardware acceleration so I'm sure it's gonna take much longer.

---

Wait, but aren't our remaining tests all unit tests without UI interactions anyway? 

I was also confused by the Actions UI claiming that all four individual jobs had passed, but the meta job was still awaiting its result. Do you know what was up with that?

---

> Wait, but aren't our remaining tests all unit tests without UI interactions anyway?

That's correct yeah. We can probably even make the new instrumentation tests work in the JVM by mocking `android.net.Uri` behavior and avoid this entire problem altogether.

> I was also confused by the Actions UI claiming that all four individual jobs had passed, but the meta job was still awaiting its result. Do you know what was up with that?

The awaiting result problem is a bug that sometimes affects matrix builds. Waiting a few extra seconds usually resolves it for me.

---

> The awaiting result problem is a bug that sometimes affects matrix builds. Waiting a few extra seconds usually resolves it for me.

17 hours in and counting... Well, but Ubuntu broke this for API level 29 anyway, so I will revert this change.